Cruise Ship Function

wooyunlong opened the pull request:
-Add Cruise Ships section to the home page, below Flights section.
-Cruise ship data April 16.

freddiewanah commented on src/App.js:
```diff
@@ -291,8 +291,17 @@ function App() {
           </Grid>
           <Grid item xs={12} className="removePadding">
-            <Navbar setNav={setNav} nav={nav} />
-            {/*<Navbar  province={province} overall={overall} myData={myData} area={area} data={data} setProvince={setProvince} gspace={gspace} columns={columns}/>*/}
```
should change back

wooyunlong replied:
I have changed it back

freddiewanah commented on src/Header.js:
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
this is not necessary ?

wooyunlong replied:
My branch was a bit out of date, so it remained some old files that's already deleted from the latest dev branch. The merge from Dev didn't remove such files from my branch. I have removed them manually.

freddiewanah commented on src/Homepage.js:
```diff
@@ -0,0 +1,466 @@
+              newData={myData}
+            />
+            {/*{*/}
+            {/*province ? false :*/}
```
remove?

wooyunlong replied:
My branch was a bit out of date, so it remained some old files that's already deleted from the latest dev branch. The merge from Dev didn't remove such files from my branch. I have removed them manually.

wooyunlong commented:
The branch was a bit out of date, so some old files (deleted from the latest dev branch) are still there. I have reworked the files on my branch based on the latest dev, so now only 3 files on the pr have been changed. A brand new Ships.js. A homepage.js, I deleted 3 imports that's not being used anymore. A .gitignore file that I accidentally committed when I opened pr, and I changed it back.
